osclib/conf.py

```diff
@@ -58,6 +60,8 @@
         'openqa': 'https://openqa.opensuse.org',
         'lock': 'openSUSE:%(project)s:Staging',
         'lock-ns': 'openSUSE',
+        'delreq-monitor': 'factory-maintainers',
```

Comment:
Would it make sense to use a different user/group like you originally did to avoid the continued mixing of `factory` in `non-factory` projects?

Reply:
the idea here was that 'the one that will have to accept it into oS:F in the end must be in factory-maintainers' anyway; and the 'last step', removing the review of the 'delreq-monitor', would happen directly in the ```osc staging accept``` call; so definitively by a user in the factory-maintainers group.

If we take a 2nd group, it _must_ have all the member of factory-maintainers, plus potentially more (which did not seem to realistic)

Reply:
So I guess this is a symptom of the larger problem that that group is also used for Leap already. As such it makes sense to use in this context, but should be considered for changing for Leap in general at some point.

Reply:
indeed - so fo far the group allowed to write into the product directly was small and the same set of users is used across TW and Leap (to back each other up);

Reply:
So perhaps `masters-of-the-obs` :)

Reply:
That sounds more like the obs admins; happy not to be part of that (most of the times)